models/nxos/ospf_global/nxos_ospf_global.yaml

```diff
@@ -0,0 +1,529 @@
+                              type: int
+                            max_wait_time:
+                              description:
+                                - Maximum wait time between SPF calculations.
+                              type: int
+                redistribute:
+                  description:
+                    - Redistribute information from another routing protocol.
+                  type: dict
+                  suboptions:
+                    protocol:
+                      description:
+                        - The name of the protocol.
+                      type: str
+                      choices: ['bgp', 'direct', 'eigrp', 'isis', 'lisp', 'ospf', 'rip', 'static']
+                    id:
+                      description:
+                        - The identifier for the protocol specified.
+                      type: str
+                    route_map:
+                      description:
+                        - The route map policy to constrain redistribution.
+                      type: str
+                    maximum_prefix:
+                      description:
+                        - Maximum number of IP prefixes redistributed.
+                      type: int
+                    maximum_paths:
+                      description:
+                        - Maximum paths per destination.
+                      type: int
+                distance:
+                  description:
+                    - Configure the OSPF administrative distance.
+                  type: int
+                default_information:
+                  description:
+                    - Control distribution of default routes.
+                  type: dict
+                  suboptions:
+                    originate:
+                      description:
+                        - Distribute a default route.
+                      type: dict
+                      suboptions:
+                        enable:
+                          description: 
+                            - Enable distribution of default route.
+                          type: bool
+                        always:
+                          description:
+                            - Always advertise a default route.
+                          type: bool
+                        route_map:
+                          description:
+                            - Policy to control distribution of default routes
+                          type: str
+                default_metric:
+                  description:
+                    - Specify default metric for redistributed routes.
+                  type: str
+                areas:
+                  description:
+                    - Configure properties of OSPF Areas.
+                  type: list
+                  elements: dict
+                  suboptions:
+                    area_id:
+                      description:
+                        - The Area ID as an integer or IP Address.
```

Comment:
The id is not an ip address as such. It is represented in the same form as an ip address - 32 bit number expressed in decimal or dotted-decimal notation.
Are we going to have this key as area_id or number?

Reply:
@GomathiselviS This is how the device shows this option:
```
nxos-9k-rdo(config-router)# area ?
  A.B.C.D or <0-4294967295>  Area Id as an integer or ip address
```